packages/common/src/audius-query/createApi.ts

```diff
@@ -398,7 +398,7 @@ const fetchData = async <Args, Data>(
       // Format entities before adding to cache
       entities[Kind.USERS] = mapValues(
         entities[Kind.USERS] ?? [],
-        (user: UserMetadata) => reformatUser(user, audiusBackend)
```

Comment:
Every time you remove an audiusBackend reference, a forest sprite gets its wings.